sdk/servicebus/Azure.ResourceManager.ServiceBus/tests/Tests/ServiceBusNamespaceTests.cs

```diff
@@ -173,8 +174,8 @@ public async Task GetPrivateLinkResources()
             ServiceBusNamespace serviceBusNamespace = (await namespaceCollection.CreateOrUpdateAsync(namespaceName, new ServiceBusNamespaceData(DefaultLocation))).Value;
 
             //get private link resource
-            IReadOnlyList<PrivateLinkResource> privateLinkResources = (await serviceBusNamespace.GetPrivateLinkResourcesAsync()).Value;
-            Assert.NotNull(privateLinkResources);
+            var privateLinkResources = serviceBusNamespace.GetPrivateLinkResourcesAsync().EnsureSyncEnumerable().ToList();
+            Assert.IsNotEmpty(privateLinkResources);
```

Comment:
This is the only test that fails, cause `privateLinkResources` collection is empty.

Reply:
thats expected, I think we can do a .FirstOrDefault() and assert it doesn't throw.